src/main/java/com/raisetech10/liquor_management/service/LiquorServiceImpl.java

```diff
@@ -22,11 +24,26 @@ public LiquorServiceImpl(LiquorMapper liquorMapper) {
     public List<Liquor> findAll(){
         return liquorMapper.findAll();
     }
+
+    //GET Excepion
```

Comment:
Typo（タイポ）してますね！
```suggestion
    //GET Exception
```
他にも、今回のPRの修正とは違うところで警告が出ていますので、全体的に修正しちゃいましょう！

Reply:
上記//GET Exceptionへ変更いたしました。

Reply:
Typo修正しました！